manuscript/chapter3-contributor-mode.md

```diff
@@ -172,6 +217,12 @@ What are the benefits of applying the art of documenting then?:
  * You help improving the project's branding
  * You provide useful resources for newcomers
  * You feel confident for you know the topic or you have a real experience with it
+ * You will getting a growth mindset
```

Comment:
I'd suggest to link to [Carol Dweck](https://ed.stanford.edu/faculty/dweck) or [her book](https://www.littlebrown.co.uk/titles/carol-dweck/mindset-updated-edition/9781472139962/) on that matter.